pki.spec

```diff
@@ -187,7 +187,6 @@ BuildRequires:    xerces-j2
 %if 0%{?rhel} && ! 0%{?eln}
 BuildRequires:    resteasy >= 3.0.26
 %else
-BuildRequires:    jboss-annotations-1.2-api
```

Comment:
I think we can actually simplify lines 187-196 into just:
```
BuildRequires:    resteasy >= 3.0.26
```

Reply:
Looks that way, I did so and the RPMs came out OK :+1: